From d08db32c0e3f20ce3899145b43349d1102d31e02 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Fri, 27 Apr 2018 17:30:48 +0000
Subject: [PATCH] Makefile: Rename PREFIXDIR -> PREFIX

PREFIX is commonly used in packaging to specify the path prefix for
/usr or /usr/local so switch to that over PREFIXDIR.
---
 Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index bf11509..ee7a02f 100644
--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
-PREFIXDIR := /usr/local
+PREFIX := /usr/local
 BINDIR := /bin
 MANDIR := /share/man/man8
 PROG = numatop
@@ -73,9 +73,9 @@ $(TEST_PROG): $(TEST_OBJS) $(COMMON_OBJS) $(OS_OBJS) $(ARCH_OBJS) $(TEST_ARCH_OB
 	$(ARCH_OBJS) $(TEST_ARCH_OBJS) $(LDLIBS)
 
 install: $(PROG)
-	install -m 0755 -D $(PROG) $(DESTDIR)$(PREFIXDIR)$(BINDIR)/$(PROG)
+	install -m 0755 -D $(PROG) $(DESTDIR)$(PREFIX)$(BINDIR)/$(PROG)
 	gzip -c numatop.8 > numatop.8.gz
-	install -m 0755 -D numatop.8.gz $(DESTDIR)$(PREFIXDIR)$(MANDIR)/numatop.8.gz
+	install -m 0755 -D numatop.8.gz $(DESTDIR)$(PREFIX)$(MANDIR)/numatop.8.gz
 
 clean:
 	rm -rf *.o $(ARCH_PATH)/*.o $(TEST_PATH)/*.o $(TEST_ARCH_PATH)/*.o \
-- 
2.17.0

